docs/modules/sort-standalone/pulsar2hive.md

```diff
@@ -0,0 +1,158 @@
+						"idRootPath":"/user/hive/warehouse/t_inlong_v1_0fc00000046",
+						"partitionSubPath":"/{yyyyMMdd}/{yyyyMMddHH}",
+						"hiveTableName":"t_inlong_v1_0fc00000046",
+						"partitionFieldName":"dt",
+						"partitionFieldPattern":"yyyyMMddHH",
+						"msgTimeFieldPattern":"yyyy-MM-dd HH:mm:ss",
+						"maxPartitionOpenDelayHour":8
+					},
+					{
+						"inlongGroupId":"03600000045",
+						"inlongStreamId":"",
+						"separator":"|",
+						"partitionIntervalMs":3600000,
+						"idRootPath":"/user/hive/warehouse/t_inlong_v1_03600000045",
+						"partitionSubPath":"/{yyyyMMdd}/{yyyyMMddHH}",
+						"hiveTableName":"t_inlong_v1_03600000045",
+						"partitionFieldName":"dt",
+						"partitionFieldPattern":"yyyyMMddHH",
+						"msgTimeFieldPattern":"yyyy-MM-dd HH:mm:ss",
+						"maxPartitionOpenDelayHour":8
+					},
+					{
+						"inlongGroupId":"05100054990",
+						"inlongStreamId":"",
+						"separator":"|",
+						"partitionIntervalMs":3600000,
+						"idRootPath":"/user/hive/warehouse/t_inlong_v1_05100054990",
+						"partitionSubPath":"/{yyyyMMdd}/{yyyyMMddHH}",
+						"hiveTableName":"t_inlong_v1_05100054990",
+						"partitionFieldName":"dt",
+						"partitionFieldPattern":"yyyyMMddHH",
+						"msgTimeFieldPattern":"yyyy-MM-dd HH:mm:ss",
+						"maxPartitionOpenDelayHour":8
+					},
+					{
+						"inlongGroupId":"09c00014434",
+						"inlongStreamId":"",
+						"separator":"|",
+						"partitionIntervalMs":3600000,
+						"idRootPath":"/user/hive/warehouse/t_inlong_v1_09c00014434",
+						"partitionSubPath":"/{yyyyMMdd}/{yyyyMMddHH}",
+						"hiveTableName":"t_inlong_v1_09c00014434",
+						"partitionFieldName":"dt",
+						"partitionFieldPattern":"yyyyMMddHH",
+						"msgTimeFieldPattern":"yyyy-MM-dd HH:mm:ss",
+						"maxPartitionOpenDelayHour":8
+					},
+					{
+						"inlongGroupId":"0c900035509",
+						"inlongStreamId":"",
+						"separator":"|",
+						"partitionIntervalMs":3600000,
+						"idRootPath":"/user/hive/warehouse/t_inlong_v1_0c900035509",
+						"partitionSubPath":"/{yyyyMMdd}/{yyyyMMddHH}",
+						"hiveTableName":"t_inlong_v1_0c900035509",
+						"partitionFieldName":"dt",
+						"partitionFieldPattern":"yyyyMMddHH",
+						"msgTimeFieldPattern":"yyyy-MM-dd HH:mm:ss",
+						"maxPartitionOpenDelayHour":8
+					}
+				],
+				"name":"sid_hive_inlong6th_v3",
+				"sinkParams":{
+					"hdfsPath":"hdfs://127.0.0.1:9000",
+					"maxFileOpenDelayMinute":"5",
+					"tokenOvertimeMinute":"60",
+					"maxOutputFileSizeGb":"2",
+					"hiveJdbcUrl":"jdbc:hive2://127.0.0.2:10000",
+					"hiveDatabase":"default",
+					"hiveUsername":"hive",
+					"hivePassword":"hive"
+				},
+				"type":"HIVE"
+			}
+		]
+	},
+	"errCode":0,
+	"md5":"md5",
+	"result":true
+}
+```
+
+
+### Example: conf/sid_hive_inlong6th_v3.conf
+
+```
+{
+	"sortClusterName": "hivev3-sz-sz1",
+	"sortTaskId": "sid_hive_inlong6th_v3",
+	"cacheZones": {
+		"pc_atta6th_sz1": {
+			"zoneName": "pc_atta6th_sz1",
+			"serviceUrl": "http://9.139.53.86:8080",
```

Comment:
This IP and authentication are dangerous, please replace them with others.

Reply:
fixed, thx